test/health_checker_unittest.cc

```diff
@@ -22,35 +22,30 @@ class HealthCheckerUnittest : public ::testing::Test {
     return health_checker.IsHealthy();
   }
 
-  void SetIsolationPath(const std::string& isolation_path) {
-    isolation_path_ = isolation_path;
-    std::stringstream stream(
-        "HealthCheckFile: './" + isolation_path_ + "/unhealthy'");
-    config_.ParseConfiguration(stream);
-  }
-
-  std::string isolation_path_;
-  MetadataAgentConfiguration config_;
 };
 
+static std::istringstream IsolationPathConfig(const std::string& test_name) {
```

Comment:
Forgot this one. `static` is ok within a class, but for a free-standing function, it's preferred to have it in an anonymous namespace ([style guide](https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables)).

Reply:
`static` is no longer needed, right?